### PR TITLE
Add participation commands

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/command/account_manager.rs [hunk 1]
```diff
@@ -149,9 +149,7 @@ pub async fn restore_command(
         .with_secret_manager(secret_manager)
         // Will be overwritten by the backup's value.
         .with_client_options(
-            ClientOptions::new()
-                .with_node("http://localhost:14265")?
-                .with_node_sync_disabled(),
+            ClientOptions::new().with_node("http://localhost:14265")?, // .with_node_sync_disabled(),
```
Comment (Thoralf-M) [inline src/command/account_manager.rs]:
```suggestion
            ClientOptions::new().with_node("http://localhost:14265")?, // .with_ignore_node_health(),
```

Comment (thibault-martinez) [reply src/command/account_manager.rs]: Still commented or... ?

Comment (Thoralf-M) [reply src/command/account_manager.rs]: No harm to leave it there, might be good to enable it at some point to quickly test something

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
Missing AccountManager participation methods

**Comment by thibault-martinez** [comment]
Can I do it in another PR ?

**Review by Thoralf-M — APPROVED** [review]
Not sure where it best fits, but it might be good to also put https://github.com/iota-community/treasury/blob/main/specifications/hornet-participation-plugin.md somewhere so one can read more about the details of it (maybe add it to the account manager methods in the other PR?)

**Comment by thibault-martinez** [comment]
> Not sure where it best fits, but it might be good to also put https://github.com/iota-community/treasury/blob/main/specifications/hornet-participation-plugin.md somewhere so one can read more about the details of it (maybe add it to the account manager methods in the other PR?)

Will do

**Pull request merged**